packaging/resource_suite_s3_nocache.py

```diff
@@ -2488,6 +2488,120 @@ def recursive_put_small_files_with_checksum_and_N_flag__issue_2198(self):
             s3plugin_lib.remove_if_exists(get_filename2)
             s3plugin_lib.remove_if_exists(get_filename3)
 
+    def multiple_irsync_to_s3__issue_2204(self):
```

Comment:
Consider giving a more descriptive name to this test.

Also, is it an issue that this does not begin with the word `test`? I seem to recall `unittest` looking for functions that start with `test` or `test_` in order to consider it as a test... The same goes for `recursive_put_small_files_with_checksum_and_N_flag__issue_2198` above and the other new test below.

Reply:
Good catch.  The tests run when manually called but probably don't run when executing the entire suite.  I changed the names of the tests and ran them to make sure they still work.